nrt/data/DynaFlow/VoltageDiffTargetNormalizer/solvers.par

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file can be removed with the set integrated in the .par of the nrt (with set id="Solver")